mentat/llm_api_handler.py

```diff
@@ -170,6 +175,10 @@ class Model:
     "text-embedding-ada-002": Model(
         "text-embedding-ada-002", 8191, 0.0001, 0, embedding_model=True
     ),
+    # Fine-tuned on Jan-6 2024 with `sampler-one-hundred-v1.jsonl` data
+    "ft:gpt-3.5-turbo-1106:abante::8dsQMc4F": Model(
```

Comment:
Unfortunately only abante ai org ai keys will be able to use this model so we shouldn't add it to the list.